sky/shell/tracing_controller.cc

```diff
@@ -148,6 +150,23 @@ static void BaseTraceEventCallback(base::TraceTicks timestamp,
   }
 }
 
+static void TraceThreadMetadataToObservatory() {
+  const char* name = base::PlatformThread::GetName();
+  if (name == nullptr) {
+    return;
+  }
+  Dart_SetThreadName(name);
```

Comment:
There is at least one place in the java code where we set a thread name manually with base.  Do we know if we catch that in Dart too?